src/Pagerfanta/Pagerfanta.php

```diff
@@ -498,4 +499,24 @@ private function needsToIntegerConversion($value)
     {
         return (is_string($value) || is_float($value)) && (int) $value == $value;
     }
+
+    /**
+     * Get page number of a specific item
```

Comment:
The comment here makes it sounds like you pass in one of the actual items in the data-set (for example, for pages of customers, pass in a customer entity).

Please could you change this to something like this?

```
Get page number of the item at specified position (1-based index)
```

If you could also rename `$item` to `$position`, that would help clarify things.  (`$index` would also be okay, but my concern is that sounds more like it would be 0-based).

And finally, please could you change the method name to something like `getPageNumberForItemAtPosition`?